flink-formats-kafka/flink-json-debezium/pom.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.apache.flink</groupId>
+		<artifactId>flink-formats-kafka</artifactId>
+		<version>4.1-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>flink-json-debezium</artifactId>
```

Comment:
I'm wondering if there is a significant benefit to hanving this as a separate module.

for backwards-compatibility alone we'd need to bundle it in the kafka-connector (as it was previously provided by the distribution by default), and with no additional dependencies being added I don't see a real downside.

Reply:
re: backwards compatibility

I also just realized that the Flink SQL Client bundles flink-json as well. So we can't really separate the debezium-json formats out of flink-json ... otherwise it becomes a behavior breaking change (users suddenly have to explicitly include this new module why previously it just works out of the box).

Reply:
Gordon and I discussed offline. For the purposes of finishing off the externalization and externalizing the required confluent avro, we will table this discussion to outside of this PR. I filed https://issues.apache.org/jira/browse/FLINK-31747 to track this